src/collections/_documentation/workflow/integrations/global-integrations.md

```diff
@@ -399,7 +399,7 @@ Sentry owner or manager permissions, and GitHub owner permissions are required t
 
     [{% asset github-global-add-repo.png %}]({% asset github-global-add-repo.png @path %})
 
-GitHub should now be enabled for all projects under your Sentry organization.
+GitHub should now be enabled for all projects under your Sentry organization. Note that you can only have one GitHub organization integrated with one Sentry organization, but you can have multiple GitHub organizations connect with one Sentry organization.
```

Comment:
This reads contradictory to me.

- First sentence says [GH org, Sentry org] _must_ have a  1:1 relationship;
- second sentence says [GH org, Sentry org] _may_ be N:1 (contradicting the first sentence).

Reply:
I see what you mean - updated to convey this better. Let me know if it's more clear.